### ISSUE TITLE
NotFoundHttpException

### ISSUE THREAD

**Issue by yamilrevilla** [spatie/laravel-feed#10]
Hi guys, 
Great packages you got. Thanks for that. 

I'm having a little problem with the route, Laravel throws me a NotFoundHttpException when I enter in http://domain.local/rss

Aditionally when I include the "@include('laravel-feed::feed-links')" the code generated is 

`<link rel="alternate" type="application/rss+xml" href="http://domain.local/%5Crss" title="Test Millenium RSS">`

My config file:

```php
return [

    'feeds' => [
        [
            /*
             * Here you can specify which class and method will return
             * the items that should appear in the feed. For example:
             * '\App\Model@getAllFeedItems'
             */
            'items' => 'YARR\Properties@getAllFeedItems',

            /*
             * The feed will be available on this url.
             */
            'url' => '/rss',

            'title' => 'Test Millenium RSS',
        ],
    ],

];
```

In routes.php

```php
Route::get('/', ['as' => 'home', 'uses' => '\YARR\Http\Controllers\Frontend\IndexController@index']);
    Route::feeds();  
    Route::get('/registrarme', ['as' => 'my.register', 'uses' => '\YARR\Http\Controllers\Backend\AuthController@register']);
    Route::post('/registrarme', ['as' => 'my.post.register', 'uses' => '\YARR\Http\Controllers\Backend\AuthController@registerUser']);
    Route::post('/suscribirse', ['as' => 'my.suscribe', 'uses' => '\YARR\Http\Controllers\Backend\AuthController@suscribe']);
    Route::get('/eliminar-suscripcion/{email}', ['as' => 'my.unsuscribe', 'uses' => '\YARR\Http\Controllers\Backend\AuthController@unsuscribe']);

. . .
```

And the model:

```php
public function getFeedItemId()
    {
        return $this->id;
    }

    public function getFeedItemTitle() : string
    {
        return $this->title;
    }

    public function getFeedItemSummary() : string
    {
        return str_limit($this->description, 150);
    }

    public function getFeedItemUpdated() : Carbon
    {
        return $this->updated_at;
    }

    public function getFeedItemLink() : string
    {
        return route('property.view', [
            str_slug($this->property_type->name),
            str_slug($this->state->name),
            str_slug($this->city->name),
            str_slug($this->title),
            $this->id
        ]);
    }

    public function getFeedItemAuthor() : string
    {
        return !is_null($this->user) ? $this->user->name : 'Test Millenium';
    }

    public function getAllFeedItems() : Collection
    {
        return $this->where('status', '=', 1)->where('available', '=', 1)->get();
    }
```

Of course I placed the Spatie\Feed\FeedServiceProvider::class, after the YARR\Providers\RouteServiceProvider::class,

I'm doing something wrong?

**Comment by freekmurze** [comment]
Did you add `Route::feeds();` in your routes file?

**Comment by yamilrevilla** [comment]
Hi @freekmurze, yes, if you look my previous message you could see it.
I included the macro at the top of the file as the documentation says

**Comment by freekmurze** [comment]
Mmm, strange... I don't see anything wrong with your code. Could you try installing the package in a fresh copy of Laravel?

**Comment by yamilrevilla** [comment]
I think I found the problem.

I list all my routes using `$list = Route::getRoutes()`  and I saw that the rss uri was being generated like this:: _http://domain.local\rss_ so looking at the **FeedServiceProvider.php** file  I find this specific line:

```php
$separator = starts_with($feedConfiguration['url'], DIRECTORY_SEPARATOR) ? '' : DIRECTORY_SEPARATOR;
```
I replaced with this:
```php
$separator = '/';
```

And everything work. I'm working on Windows so that's maybe the problem Sir.
What do you think?

**Comment by freekmurze** [comment]
Yup, that's indeed it. `DIRECTORY_SEPARATOR` isn't needed there. Thanks for finding this bug.  I'll tag a fix soon.

**Comment by freekmurze** [comment]
Fixed in `1.0.8`
